modules/issue_tracker/templates/issue_tracker_controlpanel.tpl

```diff
@@ -1,13 +1,12 @@
 <h3>Navigation</h3>
 <ul>
-    {if $issue.backURL}
-        <li><a href="{$issue.backURL}">
-                    <span class="text-default">
-                        <span class="glyphicon glyphicon-backward"></span>&nbsp;Back to list
-                    </span>
-            </a>
-        </li>
-    {/if}
+    <li>
+        <a href="{$issue.backURL}">
```

Comment:
this still needs the if, but from what I can tell issue.backURL is always false now, since it was removed from the template, which means the `href=` will always be wrong